### PR TITLE
Change "Refresh" to "Edit" TOTP

### PR DESCRIPTION
**Pull request by slightlyskepticalpotat**
Closes https://github.com/DMOJ/online-judge/issues/1839, just cleaning some of my issues opened long ago

### CODE REVIEW

**Review by Xyene — COMMENTED** [review]
LGTM, but please edit your commit title to match the formatting of other titles, e.g.

image

**Review by Xyene — APPROVED** [review]
LGTM, thanks!

**Comment by quantum5** [comment]
@Xyene why did you merge this? Editing `.po` files manually like this is fundamentally broken since it just gets out of sync with crowdin and will be wiped out.

**Comment by kiritofeng** [comment]
Also isn't https://github.com/DMOJ/online-judge/pull/2269/files#diff-643b1e64ddd6a6bf50ff74bb6a83c23d40faca036dd3f208124aa4ec85c2d888R5511 just wrong anyway...

File: locale/ca/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -5508,7 +5508,7 @@ msgid "Total:"
 msgstr "Total:"
 
 #: templates/submission/list.html:350
-msgid "You were disconnected. Refresh to show latest updates."
+msgid "You were disconnected. Edit to show latest updates."
```
Comment (int-y1) [inline locale/ca/LC_MESSAGES/django.po:5511]: this change is wrong.
why are you editing the po files manually?

**Comment by int-y1** [comment]
we should merge #2142 soon to cover up these po changes

**Comment by quantum5** [comment]
> we should merge #2142 soon to cover up these po changes

No, that'll just create a conflict or not do anything. This change is bogus and has been reverted.

**Pull request merged**